File: Clg_Placement/Java/_10.DisplayLongestName.java
class GFG {
    String longest(String names[], int n) {
        int ind = -1;
        int longest = -1;
        for(int i=0;i<n;i++){
            int length = names[i].length();
            if(length>longest){
                longest = length;
                ind = i;
            }
        }
        return names[ind];
    }
}

File: Clg_Placement/C++/Arrays/_10.DisplayLongestName.cpp
class Solution{
    public:
    string longest(string names[], int n)
    {
        pair<int,int>longest={-1,-1};
        for(int i=0;i<n;i++){
            int length = names[i].length(); //can also use .size() or strlen()
            if(length>longest.second){
                 longest.second=length;
                 longest.first = i;
            }
        }
        return names[longest.first];
    }
};
